lib/barbeque/exception_handler.rb

```diff
@@ -0,0 +1,22 @@
+require 'barbeque/configuration'
+
+module Barbeque
+  module ExceptionHandler
+    def self.handle_exception(e)
+      handler = const_get(Barbeque.config.exception_handler, false)
```

Comment:
IMO, `const_get` takes a cost so you might like to memorize `handler` like `@handler ||= const_get(Barbeque.config.exception_handler, false)`.